config/jobs/kubernetes-sigs/cluster-api-provider-kubevirt/OWNERS

```diff
@@ -0,0 +1,16 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+reviewers:
+- agradouski
+- cchengleo
+- cluster-api-admins
```

Comment:
@cchengleo I think you should remove all aliases like `cluster-api-admins`. They are not defined in any OWNERS_ALIAS file in this repo.

Reply:
Never mind. You are not yet in the k8s org it seems.

Reply:
Yeah, that's why this PR is not progressing.